apps/server/src/shared/infra/identity-management/keycloak-administration/service/keycloak-administration.service.spec.ts

```diff
@@ -61,7 +46,14 @@ describe('KeycloakAdministrationService', () => {
 		settings = module.get(KeycloakSettings);
 		service = module.get(KeycloakAdministrationService);
 
+		kcAdminClient.realms.update = jest.fn();
```

Comment:
Why do you have to set that explicitly?

Reply:
Because ts-jest library only mocks all methods on an object and not the properties. Therefore properties must be mocked additionally if they contain a function.